### PR TITLE
[DOC] update dev FAQ based on starter kit

### LINKED ISSUE #51: Link to BIDS Apps paper, link to "BIDS Apps" BEP027 (aka "BIDS Apps spec"?)
I didn't find a link to either of those two important resources:

- [x] https://doi.org/10.1371/journal.pcbi.1005209
- [x] https://bids.neuroimaging.io/bep027

### PR DESCRIPTION
**Pull request by Remi-Gau**
closes #51 

to be merged after #42

### CODE REVIEW

**Review by sappelhoff — COMMENTED** [review]
still marked as WIP - please let me know if you want to have a review and/or merge @Remi-Gau

**Comment by Remi-Gau** [comment]
> still marked as WIP - please let me know if you want to have a review and/or merge @Remi-Gau

soon_o_815840

File: css/main.css [hunk 1]
```diff
@@ -46,6 +46,14 @@ pre.highlight code,
...
 }
 
+.post h2 {
+  font-size: 150%;
+}
+
+.post h3 {
+  font-size: 125%;
+}
+
```
Comment (Remi-Gau) [inline css/main.css:56]: The headings were too big to my liking.

Have not checked how things look if one starts using `h4` or `####` for headings

File: dev_faq.md [hunk 2]
```diff
@@ -3,6 +3,118 @@ layout: post
...
+
+```bash
+$ bids_dir=/home/jane/input/ds00101
+
+$ output_dir=/home/jane/output/
+
+$ analysis_level=participant
+
+$ BIDSapp_name $bids_dir $output_dir $analysis_level
+```
```
Comment (Remi-Gau) [inline dev_faq.md:60]: This feels a bit silly, no?

File: dev_faq.md [hunk 3]
```diff
@@ -3,6 +3,118 @@ layout: post
...
+
+## API
+
+### What should the API of my BIDS App look like?
+
+The obligatory arguments of the API of any BIDS App are:
+
+- `bids_dir`
+- `output_dir`
+- `analysis_level`
```
Comment (Remi-Gau) [inline dev_faq.md:48]: FYI the original text had "participant label" as obligatory.

File: dev_faq.md [hunk 4]
```diff
@@ -3,6 +3,118 @@ layout: post
...
+docker hub will happen automatically via Circle-ci.
+
+If you want your App to show on the BIDS App website
+[here](http://bids-apps.neuroimaging.io/apps/)), you will in any case have to
+update the `_config.yml` in the
+[BIDS App website repository](https://github.com/BIDS-Apps/bids-apps.github.io.git).
+
+<!--
+TODO: how do I "submit" an App? Is there a review process? What are the requirements?
+-->
```
Comment (Remi-Gau) [inline dev_faq.md:110]: planning this for another PR

**Comment by Remi-Gau** [comment]
@gkiar if you have a bit of time to check this.

It's liikely there will be more content in the dev FAQ added here.

Part could come from BEP27 once finalized or from specifying a review process for BIDS apps.

File: dev_faq.md (hunk 2 above)
Comment (sappelhoff) [reply dev_faq.md:60]: agreed, using the $variables only complicates this example

**Comment by gkiar** [comment]
Thanks for the tag! I'll do my best to take a look in the next week or so. Please don't be shy to re-ping if I'm tardy :sweat_smile:

**Comment by Remi-Gau** [comment]
If you are too tardy we might merge anyway but ping you when we try to add more info than "simply" transferring things that were on the starterkit

File: _includes/header.html [hunk 5]
```diff
@@ -10,7 +10,7 @@
   <div class="panel-main">
     <div class="panel-main__inner panel-inverted">
       <div class="panel-main__content">
-        <img src="/images/gear.jpg" class="user-image" alt="My Profile Photo" />
+        <img src="images/BIDS_apps_logo_black.jpg" class="user-image" alt="My Profile Photo" />
```
Comment (sappelhoff) [inline _includes/header.html:13]: is this maybe a confusion with https://github.com/BIDS-Apps/bids-apps.github.io/pull/56 ?

File: dev_faq.md (hunk 2 above)
Comment (sappelhoff) [reply dev_faq.md:60]: on the other hand, I am also fine with it as is ... no biggie

File: dev_faq.md [hunk 6]
```diff
@@ -3,6 +3,118 @@ layout: post
...
+$ BIDSapp_name $bids_dir $output_dir $analysis_level
+```
+
+### Can I add more arguments to the API of my App?
+
+Every BIDS App must use the mandatory arguments mentioned above, but you are
+free to add more that are specific to the task your App will perform.
+
+We recommend you follow the guidelines mentioned in the
+["execution" BIDS extension proposal](https://docs.google.com/document/d/104HLZedFtx0TaXEUwd7eyWvJUlc0CcSUtCzwjNgmGxE)
```
Comment (sappelhoff) [inline dev_faq.md:68]:
```suggestion
[BIDS extension proposal 027](https://bids.neuroimaging.io/bep027)
```

**Review by sappelhoff — APPROVED** [review]
just some minor comments, I think this can be merged if those are addressed :+1:

File: _includes/header.html (hunk 5 above)
Comment (Remi-Gau) [reply _includes/header.html]: you are right, I mixed up my PRs

File: dev_faq.md (hunk 2 above)
Comment (Remi-Gau) [reply dev_faq.md]: updated it.

**Comment by sappelhoff** [comment]
Thanks @Remi-Gau

**Pull request merged**